src/Runner.jl

```diff
@@ -415,6 +421,18 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
         if Sys.isfreebsd(p)
             add_system_includedir(flags)
         end
+
+        if !Sys.isbsd(p)
+            gcc_version, llvm_version = select_compiler_versions(p, compilers)
+            append!(flags, String["-I/opt/$(aatriplet(p))/$(aatriplet(p))/include/c++/$(gcc_version)",
+                                  "-I/opt/$(aatriplet(p))/$(aatriplet(p))/include/c++/$(gcc_version)/$(aatriplet(p))",
+                                  "-I/opt/$(aatriplet(p))/$(aatriplet(p))/include/c++/$(gcc_version)/backward",
+                                  "-I/opt/$(aatriplet(p))/lib/gcc/$(aatriplet(p))/$(gcc_version)/include",
+                                  "-I/opt/$(aatriplet(p))/lib/gcc/$(aatriplet(p))/$(gcc_version)/include-fixed",
+                                  "-I/opt/$(aatriplet(p))/$(aatriplet(p))/include",
+                                  "-I/opt/$(aatriplet(p))/$(aatriplet(p))/sys-root/include"])
+        end
```

Comment:
Why this instead of `--gcc-toolchain`?  Did you see #247 and #248?  One problem of this approach is also that this triggers `-Werror` failures in case some paths are unused, which isn't great.

Reply:
gcc-toolchain wasn't reliable for me when testing, our gcc install is a bit weird because we set a sysroot that isn't the usual one and clang isn't able to find stuff. Specially in mingw.
About the warning, given that it didn't complain when compiling very simple C code I would be surprised if it did for more complicated ones?

Reply:
I'm pretty sure I've seen that warning in the wild.

Reply:
I discovered why it might complain, we should use -isystem here. Those if unused do not get warnings.

Reply:
Ah, that's probably a good idea here